Makefile

```diff
@@ -0,0 +1,60 @@
+.PHONY: dummy_translations extract_translations fake_translations help pull_translations push_translations \
+
+
+.DEFAULT_GOAL := help
+
+help: ## display this help message
+	@echo "Please use \`make <target>' where <target> is one of"
+	@perl -nle'print $& if m{^[a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}'
+
+clean: ## remove generated byte code, coverage reports, and build artifacts
+	find . -name '__pycache__' -exec rm -rf {} +
+	find . -name '*.pyc' -exec rm -f {} +
+	find . -name '*.pyo' -exec rm -f {} +
+	find . -name '*~' -exec rm -f {} +
+	rm -fr build/
+	rm -fr dist/
+	rm -fr *.egg-info
+
+quality: ## check coding style with pycodestyle and pylint
+	echo 'TODO: Refactor quality checks from .travis.yml into this Makefile'
+	exit 1
+
+requirements: ## install development environment requirements
+	pip install -r requirements.txt --exists-action w
+	pip install -r requirements-dev.txt --exists-action w
+
+test: clean ## run tests in the current virtualenv
+	python run_tests.py
+
+selfcheck: ## check that the Makefile is well-formed
+	@echo "The Makefile is well-formed."
+
+## Localization targets
+
+extract_translations: ## extract strings to be translated, outputting .po files
+	rm -f locale/en/LC_MESSAGES/django.po
+	django-admin.py makemessages -l en -d django
```

Comment:
Not sure where django-admin and i18n_tool are coming from?

Reply:
It's installed via `$ make requirements` which installs `requirements-dev.txt`. Those requirements are a standard requirement for Open edX related applications.